pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -326,7 +326,7 @@ type WorkflowSpec struct {
 	// Host networking requested for this workflow pod. Default to false.
 	HostNetwork *bool `json:"hostNetwork,omitempty" protobuf:"bytes,14,opt,name=hostNetwork"`
 
-	// Set DNS policy for the pod.
+	// Set DNS policy for the pod and argo agent.
```

Comment:
```suggestion
	// Set DNS policy for workflow pods.
```

For consistency, the rest of the descriptions say either "workflow pods" or "all pods in the workflow", both of which would include the agent

Reply:
Thanks！changed it